### DISCUSSION TITLE
Additional tree icons: 🌳 and 🌲

### DISCUSSION THREAD

**Discussion by taimar** [lucide-icons/lucide#682] [Ideas]
We already have 🌴 but this can be inappropriate in many contexts.
Would be nice to have 🌳 and 🌲 as well.

Links in the same order just in case there are rendering issues with the emojis:
https://emojipedia.org/palm-tree/
https://emojipedia.org/deciduous-tree/
https://emojipedia.org/evergreen-tree/

Thank you!

**Comment by karsa-mistmere** [comment]
There's an ongoing issue (#599) as well as a separate pull request (#644), I really don't think we need a discussion as well. :)

**Reply by taimar** [reply]
Thank you for the quick reply and I’m very glad to see them already in the works!
